package.json

```diff
@@ -35,22 +35,34 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
-    "watch": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "npm run compile && node ./node_modules/vscode/bin/test"
+    "vscode:prepublish": "yarn run package",
```

Comment:
this requires yarn to be installed separatly (it's not as devDependency)

yarn these days doesn't have a compelling advantage over npm (which works OOTB), but more importantly it's lacking an equivalent to `npm audit fix`.
Unless you have convincing arguments to require yarn over npm, I say, let's keep all our extensions standardized over npm for now.

Reply:
Ya, currently there is no compelling advantage of using yarn in place of NPM,
I have reverted this change to use npm.